doc/tools/docgram/fullGrammar

```diff
@@ -2454,8 +2452,6 @@ as_or_and_ipat: [
 
 eqn_ipat: [
 | "eqn" ":" naming_intropattern
-| "_eqn" ":" naming_intropattern
-| "_eqn"
```

Comment:
This does not seem to belong here.

Reply:
This is just because `fullGrammar` and `orderedGrammar` are not automatically updated whenever the grammar is changed (for now). This is thus a follow-up of #11877. Note that these changes were automatically generated.